client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/style.scss

```diff
@@ -0,0 +1,6 @@
+.migration-key-placeholder {
+	display: block;
```

Comment:
Are we still putting these in alpha order?

Reply:
I remember we found somewhere in a PR discussion that it uses the logical order in Calypso. But I didn't find where it is now. Just found this related doc: https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#positioning. I remember @m1r0 found it. Just pinging here in case he remembers quickly where it was documented.

But I updated it here to have the `height` after the `display`: https://github.com/Automattic/wp-calypso/pull/92608/commits/ac46a0c6c8e9bf32843d12a928886e1172cb95b8

Reply:
I was surprised that we use alpha order as well. [Here's the discussion](https://github.com/Automattic/wp-calypso/pull/92215#discussion_r1661047621) from another PR with @Imran92.

Reply:
Thank you for the reference, Miro!

> I was surprised that we use alpha order as well

Do you mean you were surprised that we don't use alpha order?

Reply:
Oh, I might read that wrong. As noted in the discussion mentioned above, I prefer the properties to be in logical order instead of alphabetical. But I'm ok with both. 🙂

Reply:
I share the same opinion! And I think we should always use it since the docs say that we follow the WP standards. :)